source/Userdocs/Software/NeuroMLlite.md

```diff
@@ -1 +1,37 @@
 # NeuroMLlite
+
+NeuroMLlite is a common framework for reading/writing/generating network specifications which builds on NeuroML 2.
+
+It is intended to provide a high level specification which can be used to generate networks in NeuroML
+and many other formats, including graphical and directly in neuronal simulators
+
+## Quick start
+
+```
+git clone https://github.com/NeuroML/NeuroMLlite.git
```

Comment:
If it isn't `pip install`, I wouldn't put it in a "quickstart". We don't want end users to clone repos and build stuff from source. It makes people feel like the software isn't ready to use, and scares them away! XD